docs/dev/guidelines/client-tests.rst

```diff
@@ -252,48 +252,46 @@ Some guidelines:
+9. When expecting results use :code:`expect` for client tests. That call **must** be followed by another assertion statement like :code:`toBeTrue()`. It is best practice to use more specific expect statements rather than always expecting boolean values. It is also recommended to extract as much as possible from the `expect` statement.
 
     For example, instead of
 
     .. code:: ts
 
-        expect(course == undefined).toBe(true);
+        expect(course == undefined).toBeTrue();
         expect(courseList.length).toBe(4);
 
     extract as much as possible:
 
     .. code:: ts
 
-        expect(course).toBe(undefined);
+        expect(course).toBeUndefined();
         expect(courseList).toHaveLength(4);
 
-10. If you have minimized :code:`expect` and can choose between multiple verification functions providing the same functionality, choose the most generic one. This way we will use as few functions as possible. For example prefer :code:`toBe(true)` and :code:`toBe(false)` over :code:`toBeTrue()` and :code:`toBeFalse()`.
+10. If you have minimized :code:`expect`, use the verification function that provides the most meaningful error message in case the verification fails. You can use verification functions from core Jest <https://jestjs.io/docs/expect> or from Jest Extended <https://github.com/jest-community/jest-extended#api>.
 
-11. Use `Jest <https://jestjs.io/docs/expect>`__ whenever possible. Use `Jest Extended <https://github.com/jest-community/jest-extended#api>`_ only if it shortens the expect statements considerably and makes it more readable.
-
-12. For situations described below, only use the uniform solution to keep the codebase as consistent as possible.
+11. For situations described below, only use the uniform solution to keep the codebase as consistent as possible.
 
   +--------------------------------------------------------+-----------------------------------------------------------------+
   | Situation                                              | Solution                                                        |
   +========================================================+=================================================================+
-  | Expecting a boolean value                              | :code:`expect(value).toBe(true);`                               |
-  |                                                        |                                                                 |
-  |                                                        | :code:`expect(value).toBe(false);`                              |
+  | Expecting a boolean value                              | :code:`expect(value).toBeTrue();`                               |
+  |                                                        | :code:`expect(value).toBeFalse();`                              |
+  |                                                        | :code:`expect(value).toBeBoolean();`                            |
```

Comment:
```suggestion
```
We shouldn't just check for a type. This is bad testing behaviour in my opinion. If we know that it's a boolean we should be able to tell what boolean we expect.

Reply:
👍